vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2

```diff
@@ -1,19 +1,13 @@
 [global]
+workgroup = MYGROUP
+server string = Samba Server Version %v
+log file = /var/log/samba/log.%m
+max log size = 50
```

Comment:
I don't think want such small logs?

Reply:
Let's avoid log rotate by setting it to `0`